lib/internal/bootstrap/pre_execution.js

```diff
@@ -48,10 +48,10 @@ function prepareMainThreadExecution(expandArgv1 = false) {
   initializeClusterIPC();
 
   initializeDeprecations();
-  initializeFrozenIntrinsics();
   initializeCJSLoader();
   initializeESMLoader();
   loadPreloadModules();
+  initializeFrozenIntrinsics();
```

Comment:
This also needs to be applied to lib/internal/main/worker_thread.js.

Reply:
should be fixed / added test